packages/core/src/Serialization/Update/ProtocolParamUpdate.ts

```diff
@@ -470,14 +472,10 @@ export class ProtocolParamUpdate {
       ? UnitInterval.fromFloat(Number(parametersUpdate.decentralizationParameter))
```

Comment:
decentralizationParameter was also removed in conway era.
I am investigating this and a few other serialization issues on the conway-era branch.
I will update this PR once the implementation is stable.

Reply:
decentralizationParameter was actually removed in babbage era, but there shouldn't be a need to remove this, that way the serialization utils can be use to serialize/deserialize older transaction too. When this fields are deprecated their key is no longer used (in this case I believe is 12), but no other field can take its place, so there is no harm on leaving them there

Reply:
In my particular case (e2e test), the raw txBuilder is passing a manually built protocol params to a conway params update proposal.
I am trying to prevent this by building the typesafe version of the `fromCore` when called via conway governance.
Otherwise, the effect is that txbuilder will build the tx for you, even if those protocol params were deprecated.
But the change will not cripple the ser/des part, it will only make it more restrictive when called from the gov path

Reply:
thats a pretty good solution 🚀